### PR TITLE
Add Coinbase Wallet App Docs + additional

### PR DESCRIPTION
**Pull request by themacexpert**
### Description
This PR includes the following:
- Adds documentation for Coinbase Wallet App
- Update democracy param based on spec in runtime
- Update Moonbeam Binary release version tag + sha256 hash for v0.18.1

### Checklist

- [x] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [x] If images have been added, I have run the `compress-images.py` script to compress the images.
- [ ] If images have been moved, I have created an additional PR in `moonbeam-docs-cn` to update images to their new paths
- [x] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables

### Corresponding PRs

Please link to any corresponding PRs here.

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [x] No additional PRs are required after the translations are done

#### Items to be Updated

Please list any of the items that will need to be added or deleted after the translations are done here.

### CODE REVIEW

**Comment by eshaben** [comment]
@themacexpert Just a heads up... It looks like one of the names of the variables was changed from days to hours, so another PR will need to be opened in the moonbeam-docs-cn repo so it doesn't break the usage of it from the chinese site

File: tokens/connect/coinbase-wallet.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+!!! note
+    You can add custom tokens to your Coinbase Wallet by importing the contract address. However, these custom tokens may not be supported by Coinbase Exchange. If you send a custom token from your Coinbase Wallet to Coinbase Exchange that is not supported by Coinbase Exchange, you will lose those funds forever.  
+
+In this guide, we'll go through the process of setting up the Coinbase Wallet mobile application and configuring it for the Moonbeam Network. 
+
+## Install the Coinbase Wallet App {: #install-the-coinbase-wallet-app } 
+
+You can download the Coinbase Wallet app from the iOS App Store or the Google Play Store. Coinbase maintains [this homepage](https://wallet.coinbase.com/?_branch_match_id=977295450874474909&_branch_referrer=H4sIAAAAAAAAA8soKSkottLXT8%2FXS07SLddLzs%2FVD8%2FJynFKSy02zE4CAFZ0JzQfAAAA){target=_blank} with the download links for both.
```
Comment (eshaben) [inline tokens/connect/coinbase-wallet.md:21]: I prefer to use more descriptive link text, I think what you could do here is something like

```You can [download the Coinbase Wallet app](https://wallet.coinbase.com/?_branch_match_id=977295450874474909&_branch_referrer=H4sIAAAAAAAAA8soKSkottLXT8%2FXS07SLddLzs%2FVD8%2FJynFKSy02zE4CAFZ0JzQfAAAA){target=_blank} from the iOS App Store or the Google Play Store.```

**Review by eshaben — CHANGES_REQUESTED** [review]
I love these images!! 😍 

The index page icon is missing from `/tokens/connect/` aside from that there are a few little things but overall this is really great!!

**Comment by themacexpert** [comment]
Thanks for the review @eshaben! I cleaned up everything.

**Comment by themacexpert** [comment]
Good catch thanks @eshaben! Fixed the logo and addressed the hidden comments.

**Review by eshaben — APPROVED** [review]
LGTM :sparkles:

**Pull request merged**